### PR TITLE
Musa

### PR DESCRIPTION
**Pull request by dadyymusa**
[1]Changes made in previous db_models and entities.

[2]New models created:
    author_model.py
    staff_model.py
    student_model.py

### CODE REVIEW

File: src/db_models/order_model.py
Comment (jdxxmahmud) [inline src/db_models/order_model.py:1]: line 14?

File: src/db_models/author_model.py
Comment (jdxxmahmud) [inline src/db_models/author_model.py:1]: line 14: I don't think that's how enum should be added

File: src/db_models/orderBook_model.py
Comment (jdxxmahmud) [inline src/db_models/orderBook_model.py:1]: change table name

File: src/entities/order.py
Comment (jdxxmahmud) [inline src/entities/order.py:1]: line 11

**Review by jdxxmahmud — CHANGES_REQUESTED** [review]
Make the suggested changes

File: src/db_models/author_model.py
Comment (dadyymusa) [reply src/db_models/author_model.py:1]: not  RESOLVED

**Pull request closed without merging**